core/src/Kokkos_Atomic.hpp

```diff
@@ -215,11 +218,9 @@ extern KOKKOS_INLINE_FUNCTION void unlock_address_rocm_space(void* ptr);
 
 #include "impl/Kokkos_Atomic_Compare_Exchange_Strong.hpp"
 
-#endif  //_WIN32
-
 #include "impl/Kokkos_Atomic_Generic.hpp"
 
-#ifndef _WIN32
+//#ifndef _WIN32
```

Comment:
Why did you comment instead of removing?

Reply:
leftover because I wasn't sure this worked.